cfme/tests/cloud_infra_common/test_events.py

```diff
@@ -7,15 +7,18 @@
 from cfme.cloud.provider.gce import GCEProvider
 from cfme.control.explorer.policies import VMControlPolicy
 from cfme.infrastructure.provider import InfraProvider
-from cfme.infrastructure.provider.rhevm import RHEVMProvider
-from cfme.utils.blockers import BZ
+from cfme.infrastructure.provider.kubevirt import KubeVirtProvider
+from cfme.markers.env_markers.provider import providers
+from cfme.utils.providers import ProviderFilter
 from cfme.utils.wait import wait_for
 
 
+all_prov = ProviderFilter(classes=[InfraProvider, CloudProvider])
```

Comment:
This ProviderFilter should include the required_fields kwarg. The `gen_func` specified, `providers`, does not have a required_fields kwarg in its signature.